src/web/app/pages-student/student-profile-page/student-profile-page.component.ts

```diff
@@ -56,7 +57,7 @@ export class StudentProfilePageComponent implements OnInit {
               private statusMessageService: StatusMessageService) {}
 
   ngOnInit(): void {
-    // populate drop-down menu for nationality list
+    // populate drop-down menu for nationality and gender lists
```

Comment:
Where do we `populate drop-down menu for gender lists`?

Reply:
Sorry, intended change from previous commit. Will fix it